src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -10,10 +10,28 @@ Released 2023-Sep-05
 
 Released 2023-Aug-21
 
-* **Breaking change**: Excluded attributes corresponding to `LogRecord.EventId`,
-`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. This
-is done as the semantic conventions for these attributes are not yet stable.
-([#4781](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4781))
+* **Breaking change**: Excluded attributes corresponding to
+`LogRecord.Exception`, `LogRecord.EventId` and `LogRecord.CategoryName` from the
+exported data. See following details for reasoning behind removing each
+individual property:
+  * `LogRecord.Exception`: The semantic conventions for attributes corresponding
+    to exception data are not yet stable. Track issue
+    [#4831](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4831)
+    for details.
+  * `LogRecord.EventId`: The attributes corresponding to this property is
+    specific to .NET logging data model and there is no established convention
+    defined for it yet. Track issue
+    [#4776](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4776)
+    for details.
+  * `LogRecord.CategoryName`: The attribute corresponding to this property is
```

Comment:
Same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/4829/files#r1317857365

Reply:
this one is single attribute so this is ok.